frontend/resources/src/GenericImage.cpp

```diff
@@ -0,0 +1,515 @@
+/*
+ * GenericImage.cpp
+ *
+ * Copyright (C) 2021 by VISUS (Universitaet Stuttgart).
+ * Alle Rechte vorbehalten.
+ */
+
+
+#include "GenericImage.h"
+
+#include <algorithm>
+
+#include "glad/glad.h"
+
+#include <cstring>
+#include <list>
+
+using namespace megamol::frontend_resources;
+
+static void init_texture(unsigned int& handle) {
+    int old_handle = 0;
+    glGetIntegerv(GL_TEXTURE_BINDING_2D, &old_handle);
+
+    glCreateTextures(GL_TEXTURE_2D, 1, &handle);
```

Comment:
Does glCreateTextures modify the current texture binding, i.e., is there really a need to get and re-set the binding?

Reply:
Removed restoring old handle.